test/OpenSpaceToolkit/Astrodynamics/Access/Generator.test.cpp

```diff
@@ -1103,6 +1105,68 @@ TEST_F(OpenSpaceToolkit_Astrodynamics_Access_Generator, ComputeAccesses_5)
     }
 }
 
+TEST_F(OpenSpaceToolkit_Astrodynamics_Access_Generator, ComputeAccesses_6)
+{
+    // Ensure no interpolation is performed outside of the analysis interval
+
+    {
+        const Instant instant = Instant::J2000();
```

Comment:
Nit: clearer for the reader of the unit test to either put this instant in isoformat, or put the below `interval` instants in `instant + Duration` and `instant - Duration` format

Reply:
You want me to instantiate the J2000 epoch as an isoformat for the orbit epoch? It's not even used relative to the access interval, not sure why it matters in this case? But I advocate against instantiating the J2000 as ISOformat for sure, will lead to significant human error.